app/images/icons/transaction-shield.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M15.6836 5.06152C15.8872 4.98755 16.1128 4.98755 16.3164 5.06152L24.4414 8.01562C24.7926 8.14331 24.9999 8.45566 25 8.77734V14.9102C25 17.7584 24.131 20.3416 22.3721 22.6875C20.8016 24.782 18.8802 26.1768 16.5996 26.9121C16.211 27.0374 15.789 27.0374 15.4004 26.9121C13.1198 26.1768 11.1984 24.782 9.62793 22.6875C7.86904 20.3416 7 17.7584 7 14.9102V8.77734L7.00977 8.65723C7.05435 8.38025 7.25139 8.12731 7.55859 8.01562L15.6836 5.06152Z" stroke="#000" stroke-width="2"/>
+</svg>
```

Comment:
Icon svgs should not use strokes, this should be expanded to use a fill like all other icon svgs. It also has the wrong attributes. A good example is the `Security.svg`. Going to start a discussion in slack to check if we can use this or we absolutely need to use this for Transaction Shield feature.